.github/workflows/build.yml

```diff
@@ -262,11 +262,12 @@ jobs:
           sudo apt-get -y install build-essential binutils-dev libunwind-dev
       - name: Pin the regex dependency
         run: |
-          cd fuzz && cargo update -p regex --precise "1.9.6" --verbose && cd ..
+          cd fuzz && cargo update -p regex --precise "1.9.6" --verbose
+          cd write-seeds && cargo update -p regex --precise "1.9.6" --verbose
       - name: Sanity check fuzz targets on Rust ${{ env.TOOLCHAIN }}
         run: |
           cd fuzz
-          RUSTFLAGS="--cfg=fuzzing --cfg=secp256k1_fuzz --cfg=hashes_fuzz" cargo test --verbose --color always
+          RUSTFLAGS="--cfg=fuzzing --cfg=secp256k1_fuzz --cfg=hashes_fuzz" cargo test --verbose --color always --lib --bins
```

Comment:
Why these two additions?

Reply:
Could add comments when making changes like this. Make life easier for future devs.